docs/server-side/v2/active-rooms/get-peers.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Get Peers
+nav: 3.9
```

Comment:
Also, move this after `Retrieve a specific peer` in the sidebar. You can update this to 3.5 and bump the decimal points for Update a peer, etc.